staker-rewards/test/requirements.test.js

```diff
@@ -0,0 +1,122 @@
+import chai from "chai"
+
+import { mockMethod } from "./helpers/blockchain.js"
+import { createMockContext } from "./helpers/mock.js"
+
+import {
+  SortitionPoolAddress,
+  KeepBondingAddress,
+  BondedECDSAKeepFactoryAddress,
+} from "./helpers/constants.js"
+
+import Requirements from "../lib/requirements.js"
+import { OperatorAuthorizations } from "../lib/requirements.js"
+
+const { expect } = chai
+
+const setupContext = (context) => {
+  context.contracts.BondedECDSAKeepFactory = {
+    deployed: () => ({
+      methods: {
+        getSortitionPool: (application) => ({
+          call: async (_, blockNumber) => {
+            return SortitionPoolAddress
+          },
+        }),
+        isOperatorAuthorized: (operator) => ({
+          call: mockMethod(
+            BondedECDSAKeepFactoryAddress,
+            "isOperatorAuthorized",
+            (inputs) => inputs.operator === operator,
+            false
+          ),
+        }),
+      },
+    }),
+  }
+
+  context.contracts.KeepBonding = {
+    deployed: () => ({
+      options: {
+        address: KeepBondingAddress,
+      },
+      methods: {
+        hasSecondaryAuthorization: (operator, sortitionPool) => ({
+          call: mockMethod(
+            KeepBondingAddress,
+            "hasSecondaryAuthorization",
+            (inputs) =>
+              inputs.operator === operator &&
+              inputs.poolAddress === sortitionPool,
+            false
+          ),
+        }),
+      },
+    }),
+  }
+
+  return context
+}
```

Comment:
Non-blocking and let's not work on this right now but I'd consider putting the content of `mock.js` and `blockchain.js` in one file and doing all of this setup there. 

In this scenario, I'd only look at `test-blockchain.js` and if I'd need anything custom, I'd then be required to do the setup above.